src/sentry/replays/usecases/ingest/event_logger.py

```diff
@@ -163,6 +168,65 @@ def log_option_events(event_meta: ParsedEventMeta, project_id: int, replay_id: s
         logger.info("sentry.replays.slow_click", extra=log)
 
 
+@sentry_sdk.trace
+def log_multiclick_events(event_meta: ParsedEventMeta, project_id: int, replay_id: str) -> None:
+    for multiclick in event_meta.multiclick_events:
+        # Sample multiclick events at 0.2% rate
+        if random.randint(0, 499) >= 1:
+            continue
+
+        is_rage_multiclick = multiclick.click_count >= RAGE_CLICK_COUNT_THRESHOLD
```

Comment:
Let's not make this determination here.  We should determine if a multi-click is rage worthy at read time (using a query).